server/portal/apps/accounts/api/views/unit_test.py

```diff
@@ -1,59 +0,0 @@
-from django.test import TestCase, RequestFactory, override_settings
```

Comment:
maybe add a low priority ticket for adding unit testing /api/accounts/systems/ route?

Reply:
👍 
https://jira.tacc.utexas.edu/browse/TV3-99